content/en/smart-contracts/fundamentals/the-filecoin-virtual-machine/index.md

```diff
@@ -23,222 +23,44 @@ aliases:
 
-FVM data access control enables the creation and management of data centered decentralized-autonomous-organizations, which can govern and monetize data access and pool returns into a shared treasury.
+- FEVM Compatibility |  Are you an Ethereum / Solidity developer?  You can build the next killer app on FVM and make use of the [Filecoin Solidity library](https://docs.zondax.ch/fevm/filecoin-solidity/).  Learn more about how FVM is Ethereum runtime and solidity compatible in the next section. 
 
-### Perpetual storage
+- VM Agnostic | The FVM is built to be VM-agnostic, meaning support for other foreign VMs can be added in the near future.  Future versions of FVM can serve as a useful hypervisor enabling cross run-time invocations. 
 
-The FVM allows users to store data permanently, managed by repair and replication bots, which also benefit from Filecoin’s verifiable storage proofs.
+FVM brings user programmability to Filecoin, unleashing the enormous potential of an open data economy through various applications.  
 
-### Leasing
+### Use Cases
 
-FIL token holders can use their holdings to provide storage collateral and receive leasing fees. Community-generated reputation scores enable everyone to identify good borrowers.
+FVM Actors enable a huge range of use cases to be built on Filecoin.  Here are just a few potential examples:   
 
-### Ethereum compatibility
+- Data Access Control | FVM Actors can enable a client to grant retrieval permission for certain files to a limited set of third-party Filecoin wallet addresses.  
 
-The FVM is fully EVM-compatible, allowing new ERC-20 tokens to be launched on the Filecoin network and enabling layer two networks to leverage its storage power and security. The majority of EVM tooling can also be used within the Filecoin ecosystem.
+- DataDAO | FVM Actors can enable the creation of decentralized autonomous organizations where members govern and manage the storage, accessibility, and monetization of certain data sets and pool returns into a shared treasury.
 
-## Use-cases
+- Perpetual Storage | Because all Filecoin storage deals are time-limited, when a client makes a deal with a storage provider to store a data set with them, the client has to begin to consider whether they will want to renew this deal for the next time-period with the same storage provider or seek out other storage providers that may be cheaper.  However, FVM enables a client to automatically renew deals or find a cheaper storage provider when the time limit of a given deal has reached maturity.  This automated renewal of deals can persist, even in perpetuity, for as many cycles as can be financed by an associated endowment of FIL.  FVM Actors enable the creation and management of this endowment.
```

Comment:
Remove pipe |.

> Because all Filecoin storage deals are time-limited, when a client makes a deal with a storage provider to store a data set with them, the client has to begin to consider whether they will want to renew this deal for the next time-period with the same storage provider or seek out other storage providers that may be cheaper.

This sentence is crazy long. Simplify it.

Reply:
"All Filecoin storage deals are time-limited.  When a client makes a deal with a storage provider to store a data set with them, the client must begin to consider whether they will want to renew this deal for the next time-period with the same storage provider or seek out other storage providers that may be cheaper."